nixos/modules/virtualisation/docker.nix

```diff
@@ -40,13 +40,24 @@ in
       };
     storageDriver =
       mkOption {
-        type = types.enum ["aufs" "btrfs" "devicemapper" "overlay" "zfs"];
-        default = "devicemapper";
+        type = types.nullOr (types.enum ["aufs" "btrfs" "devicemapper" "overlay" "zfs"]);
+        default = null;
         description =
           ''
             This option determines which Docker storage driver to use.
```

Comment:
Please add to description what happens with the 'null' value. I.e., what is the (new) default storage driver?